dbt/scripts/export_models.py

```diff
@@ -107,12 +107,8 @@ def parse_args():
         logger.info("Export completed successfully.")
 
     except Exception as e:
-        # The CloudWatch log handler does not include "ERROR - " at the
-        # beginning of log lines, so we need to preface the message with
-        # "ERROR - " to trigger alarms.
-        logger.error(f"ERROR - {e}")
```

Comment:
Now that we're pushing `levelname` to CloudWatch as part of all log messages (see comments below), everything that `logger.error()` pushes will include the string `ERROR`, so we no longer need to add it manually.